### PR TITLE
Update: Available BSIPs Table

### PR DESCRIPTION
**Pull request by ryanRfox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,17 @@ Number             | Title                                                    |
 [37](bsip-0037.md) | Allow new asset name to end with a number                                   | oxarbitrage | Protocol | Installed
 [38](bsip-0038.md) | Add target collateral ratio option to short positions                       | Abit More | Protocol | Installed
 [39](bsip-0039.md) | Automatically approve proposals by the proposer                             | Fabian Schuh | Protocol | Draft
-[40](bsip-0040.md) | Custom active permission                                                    | Stefan Schießl | Protocol | Draft
+[40](bsip-0040.md) | Custom Active Permission                                                    | Stefan Schießl | Protocol | Accepted
```
Comment (pmconrad) [inline README.md]: Only milestone 1 has been accepted so far.

File: README.md [hunk 2]
```diff
@@ -47,8 +47,17 @@ Number             | Title                                                    |
...
-[44](bsip-0044.md) | Hashed Time-Locked Contract                                                 | Ryan R. Fox | Protocol | Draft
-[45](bsip-0045.md) | Introduce 'allow use as bitasset backing collateral' flag/permission to assets | Customminer | Protocol | Draft
+[43](bsip-0043.md) | Market Fee Sharing                                                          | OpenLedgerApp | Protocol | Accepted
+[44](bsip-0044.md) | Hashed Time-Locked Contract                                                 | Ryan R. Fox | Protocol | Accepted
+[45](bsip-0045.md) | Add bitAsset Backing Collateral flag/permission                             | Customminer | Protocol | Draft
+[46](https://github.com/bitshares/bsips/pull/111) | Escrow Concepts                              | Taconator | Informational | Draft
+[47](https://github.com/bitshares/bsips/pull/114) | An Explicit Voting Operation                 | Fabian Schuh | Protocol | Draft
+[48](https://github.com/bitshares/bsips/pull/115) | Add Flad to Asset to Prevent Manipulating Max Supply | Fabian Schuh | Protocol | Draft
```
Comment (pmconrad) [inline README.md]: Typo: Fla*d*

Comment (abitmore) [reply README.md]: BTW why links of 46-48 are different than the old ones?

Comment (ryanRfox) [reply README.md]: @abitmore the intent with this PR is to include all assigned BSIP entries within the table. 46+ are currently being drafted, so I referenced the relevant PR URL for adding to their discussion.  When each BSIP PR is merged, it should include the updated README reference to the published spec (else I will PR for this separately).

**Pull request merged**